gpiocontrol/source/posix/config.cmake

```diff
@@ -0,0 +1,38 @@
+#
+#  Copyright 2014-2016 CyberVision, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+# Our application name.
+set(APP_NAME demo_client)
+
+# Disable unused features
+set(WITH_EXTENSION_CONFIGURATION OFF CACHE BOOL "")
+set(WITH_EXTENSION_NOTIFICATION OFF CACHE BOOL "")
+set(WITH_EXTENSION_LOGGING OFF CACHE BOOL "")
+set(WITH_EXTENSION_CONFIGURATION OFF CACHE BOOL "")
+set(WITH_EXTENSION_USER OFF CACHE BOOL "")
+set(WITH_ENCRYPTION OFF CACHE BOOL "")
+
+# Target-independent flags.
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99  -pthread -lm  -g -Wall -Wextra")
```

Comment:
`-Werror` should only be enabled for development/CI. A different version of the compiler could easily break the build if `-Werror` is set unconditionally.